nemo/setup.py

```diff
@@ -24,7 +24,8 @@
         'tensorboardX',
         'pandas',
         'wget',
-        'onnx'
+        'onnx',
+        'sphinxcontrib-bibtex'
```

Comment:
why is there a dependency on 'sphinxcontrib-bibtex' ?

Reply:
ok, will remove it